Sources/PostgreSQLDriver/PostgreSQLConnection.swift

```diff
@@ -21,7 +21,7 @@ public final class Connection: Fluent.Connection {
     // This allows Fluent extensions to be written that can support custom
     // querying behavior.
     @discardableResult
-    public func query<E: Entity>(_ query: RawOr<Query<E>>) throws -> Node {
+    public func query<E>(_ query: RawOr<Query<E>>) throws -> Node {
```

Comment:
Does Swift 3.1/3.2 function with this change?

Reply:
Yes, as it is redundant. It is not a fix for Swift 4, just a warming fix from the new Xcode